### ISSUE TITLE
GUI is broken on KDE

### ISSUE THREAD

**Issue by Putr** [keybase/client#7783]
This is problem that has persisted since the first time I installed the app when it came out. I've seen it on multiple linux installs with KDE.

My current install from which I'm reporting: Manjaro KDE

1. Icon in the tray is ... weird (obviously broken)
2. Clicking on the icon does nothing
3. Right click on icon shows select with "show keybase", selecting that opens the "popup" menu, but not over the tray, but in the top right (where Gnome tray would be). It's also ... weird. There is no border.
4. Clicking on anything in the "popup" closes it and opens keybase in a normal window - this window is functioning as expected (as far as I can tell).

Note: Filesystem seems to be working fine.

**Comment by Putr** [comment]
Icon distortion has been reported in #3772

**Comment by heini** [comment]
Yes, this is against all KDE standards. What's a menu useful for that only has **one** entry? So clicking on the icon should simply open the main window. Right-clicking should open a (useful) menu. That strange first window that opens upon click on "Show Keybase" is just useless and should be removed altogether.

**Comment by heini** [comment]
BTW: It also doesn't obey user configuration reg. window buttons, see #17500.
